### PR TITLE
Adds support for google drive backups

### PR DESCRIPTION
**Pull request by ConorOkus**
Depends on - https://github.com/photon-sdk/photon-lib/pull/22

### CODE REVIEW

**Comment by tanx** [comment]
@ConorOkus I think this is the most up to date PR for the app correct? Would you mind updating this to use `photon-lib@ 0.8.0` which includes your GDrive PR?

File: src/action/index.js [hunk 1]
```diff
@@ -15,6 +16,9 @@ when(
       nav.reset('PinCheck');
       return;
     }
+    if (Platform.OS === 'android') {
+      await backup.authenticate();
+    }
```
Comment (tanx) [inline src/action/index.js]: This check seems redundant as there is already a check in `cloudStore.js`

```suggestion
    await backup.authenticate();
```

File: src/action/index.js [hunk 2]
```diff
@@ -27,6 +31,9 @@ when(
 when(
   () => store.walletReady,
   async () => {
+    if (Platform.OS === 'android') {
+      await backup.authenticate();
+    }
```
Comment (tanx) [inline src/action/index.js]:
```suggestion
    await backup.authenticate();
```

File: src/action/index.js [hunk 3]
```diff
@@ -5,6 +5,7 @@ import * as nav from './nav';
 import * as wallet from './wallet';
 import * as backup from './backup';
 import * as userId from './user-id';
+import {Platform} from 'react-native';
```
Comment (tanx) [inline src/action/index.js]:
```suggestion
```

**Review by tanx — CHANGES_REQUESTED** [review]
Sweet. Just made a few suggestions below.

File: src/action/index.js [hunk 4]
```diff
@@ -27,6 +28,7 @@ when(
 when(
   () => store.walletReady,
   async () => {
+    await backup.authenticate();
```
Comment (tanx) [inline src/action/index.js]: Look like we can remove this since we already authenticate above?

```suggestion
```

**Review by tanx — CHANGES_REQUESTED** [review]

File: src/action/index.js (hunk 4 above)
Comment (ConorOkus) [reply src/action/index.js]: Yeah good spot!

**Comment by tanx** [comment]
Nice job! Merging 🏖

**Pull request merged**